.eslintrc.js

```diff
@@ -56,6 +56,7 @@ module.exports = {
     '@typescript-eslint/no-unsafe-member-access': 'off',	
     '@typescript-eslint/no-unsafe-call': 'off',	
     '@typescript-eslint/unified-signatures': 'off',
+    '@typescript-eslint/no-empty-function': 'off',
```

Comment:
I turned this off, since I couldn't think of a scenario where it usefully catches anything. Maybe if this becomes a _really_ active repo one day, and we get a lot of pull requests where people accidentally forget to implement their functions or something?? For now, it's unnecessarily noisy when defining a function like `const noop = () => {}` .